idb/postgres/postgres.go

```diff
@@ -935,20 +935,20 @@ func (db *IndexerDb) yieldTxnsThreadSimple(ctx context.Context, rows pgx.Rows, r
 					err = fmt.Errorf("error decoding roottxn, err: %w", err)
 					row.Error = err
 				}
-			} else {
-				// Root transaction.
-				row.Txn = new(transactions.SignedTxnWithAD)
-				*row.Txn, err = encoding.DecodeSignedTxnWithAD(txn)
-				if err != nil {
-					err = fmt.Errorf("error decoding txn, err: %w", err)
-					row.Error = err
-				}
 			}
+			// Root transaction.
```

Comment:
Return the itxn and root txn when reading from the db if possible.

Reply:
Is there ever a scenario where we want the root txn and the inner txn?

I think we should add an option to the transaction parameters to select which we want. It would be a faster query and require less data.